lua/astrocommunity/colorscheme/solarized-osaka-nvim/init.lua

```diff
@@ -0,0 +1,4 @@
+return {
+  "craftzdog/solarized-osaka.nvim",
+  opts = {}
```

Comment:
```suggestion
  opts = {},
```

missing `,` for `stylua`